lib/ret_web/controllers/api/v1/bot_controller.ex

```diff
@@ -0,0 +1,143 @@
+defmodule RetWeb.Api.V1.BotController do
+  use RetWeb, :controller
+
+  alias RetWeb.Api.V1.{HubController}
+  alias Ret.{Hub}
+
+  @slack_api_base "https://slack.com"
+  @help_prefix "Hi! I'm the Hubs bot. I connect Slack channels with rooms on Hubs (https://hubs.mozilla.com/). Type `/hubs help` for more information."
+  @help_text "Command reference:\n\n" <>
+               "🦆 `/hubs` - Shows general information about the Hubs integration with the current Slack channel.\n" <>
+               "🦆 `/hubs help` - Shows this text you're reading right now.\n" <>
+               "🦆 `/hubs create` - Creates a default Hubs room and puts its URL into the channel topic. " <>
+               "Rooms created with `/hubs create` will inherit moderation permissions from this Slack channel and only allow Slack users in this channel to join the room.\n" <>
+               "🦆 `/hubs create [environment URL] [name]` - Creates a new room with the given environment and name, and puts its URL into the channel topic. " <>
+               "Valid environment URLs include GLTFs, GLBs, and Spoke scene pages.\n" <>
+               "🦆 `/hubs remove` - Removes the room URL from the topic and stops bridging this Slack channel with Hubs."
+
+  def create(conn, params) do
+    %{
+      # "/hubs"
+      "command" => command,
+      "channel_id" => channel_id,
+      # "help/create/etc. arg1 arg2 arg3"
+      "text" => text,
+      "team_id" => team_id
+    } = params
+
+    # TODO check if there's any double spaces
+    args = String.split(text, " ")
+    handle_command(List.first(args), channel_id, args, team_id)
+
+    conn
+    |> send_resp(200, "")
+  end
+
+  defp handle_command("help" = _command, channel_id, _args, _team_id) do
```

Comment:
nit: If you didn't know, the `= _command` isn't actually necessary.
```suggestion
  defp handle_command("help", channel_id, _args, _team_id) do
```